updates/1.082/170921_userchatsettings.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<update xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.ofw.fi/xml/2011/java-xmldb-updater/UpdaterSchema.xsd">
+  
+  <sql>
+    create table ActivityLog (
+      id bigint not null auto_increment,
+      userEntityId bigint not null,
+      type integer not null,
+      timestamp datetime not null,
+      workspaceId bigint null default null,
+      contextId bigint null default null,
+      primary key (id)
+    );
+  </sql>
+  
```

Comment:
A couple of indexes that are probably going to be useful could be added:

`<sql>create index ind_activitylog_user on ActivityLog (userEntityId);</sql>`

`<sql>create index ind_activitylog_workspace on ActivityLog (workspaceId);</sql>`